.github/workflows/build-action.yml

```diff
@@ -8,7 +8,53 @@ on:
   workflow_dispatch: {}
 
 jobs:
+  Prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      test_count: ${{ steps.count_tests.outputs.test_count }}
+      chunk_count: 8 # This is hardcoded to 8, but it can be changed to any number.
```

Comment:
what does this number do?

Reply:
Amount of chunks unit tests split into I believe.